sos/report/plugins/__init__.py

```diff
@@ -2007,8 +2007,11 @@ def _add_cmd_output(self, **kwargs):
             kwargs['priority'] = 10
         if 'changes' not in kwargs:
             kwargs['changes'] = False
-        if self.get_option('all_logs') or kwargs['sizelimit'] == 0:
+        if (not getattr(SoSCommand(**kwargs), "snap_cmd", False) and
+           (self.get_option('all_logs') or kwargs['sizelimit'] == 0)):
             kwargs['to_file'] = True
+        if "snap_cmd" in kwargs:
+            kwargs.pop("snap_cmd")
```

Comment:
Why dropping the argument (and not drop any other)? I understand we wont use the argument further more - but I am juts curious why we dont the same for other arguments, then :)

Reply:
we drop it because the argument is passed to the other functions, and `pylint` complains that the argument is not being used, so we drop it, once we use it

Reply:
I see, makes sense.